straxen/plugins/pulse_processing.py

```diff
@@ -199,6 +206,42 @@ def compute(self, raw_records, start, end):
                     pulse_counts=pulse_counts,
                     veto_regions=veto_regions)
 
+    
+@export
+@strax.takes_config(
+    strax.Option(
+        'n_tpc_pmts_he',track=False,default=800
+        ),
+    strax.Option(
+        'pulse_processing_parent_version',track=True,default=PulseProcessing.__version__
+        ),
+    *HITFINDER_OPTIONS_he)
+class PulseProcessingHe(PulseProcessing):
+    __version__ = '0.0.1'
+    provides = ('records_he', 'pulse_counts_he')
+    data_kind = {k: k for k in provides}
+    rechunk_on_save = immutabledict(
+        records_he=False,
+        pulse_counts_he=True)
+    depends_on = 'raw_records_he'
+    parallel = 'process'
+    compressor = 'lz4'  
+    
+    def infer_dtype(self):
+        dtype=dict()
+        dtype['records_he'] = strax.record_dtype(strax.DEFAULT_RECORD_LENGTH)
```

Comment:
Shouldn't this be something like
```python
record_length = strax.record_length_from_dtype(
            self.deps['raw_records_he'].dtype_for('raw_records_he'))
dtype['records_he'] = strax.record_dtype(record_length )
```

Reply:
No, thats another way of doing the same

Reply:
okay but what if the record-length isn't the default?

Reply:
ok I propose something in https://github.com/XENONnT/straxen/pull/176 to override this if we are doing some special run